database/pgx/pgx_test.go

```diff
@@ -679,5 +751,44 @@ func Test_computeLineFromPos(t *testing.T) {
 			run(true, true)
 		})
 	}
+}
 
+func Test_quoteIdentifier(t *testing.T) {
+	testcases := []struct {
+		migrationsTableQuoted bool
+		migrationsTable       string
+		expected              string
+	}{
+		{
+			false,
+			"schema_name.table_name",
+			"\"schema_name.table_name\"",
+		},
+		{
+			false,
+			"table_name",
+			"\"table_name\"",
+		},
+		{
+			true,
+			"\"schema_name\".\"table.name\"",
+			"\"schema_name\".\"table.name\"",
+		},
+		{
+			true,
+			"\"table.name\"",
```

Comment:
I think this should be `table_name` instead of `table.name`

Reply:
Fixed.